nixos/modules/services/web-apps/cloudlog.nix

```diff
@@ -507,8 +507,11 @@ in
           "d ${cfg.dataDir}/uploads        0750 ${cfg.user} ${group} - -"
           "d ${cfg.dataDir}/backup         0750 ${cfg.user} ${group} - -"
           "d ${cfg.dataDir}/logbook        0750 ${cfg.user} ${group} - -"
+          "d ${cfg.dataDir}/assets         0750 ${cfg.user} ${group} - -"
```

Comment:
I'm not familiar with `tmpfiles.d`, but I noticed that without this line the system generated only the `assets` folder and not the necessary nested folders on the following lines. With this extra line we have the desired outcome.